webcomponents/web-components/Santa/santa.stories.js

```diff
@@ -0,0 +1,22 @@
+import { document, console } from 'global';
+import { storiesOf } from '@storybook/html';
+
+// import component
+import './santa';
+import readme from './readme.md';
+
+storiesOf('Data Input|Select box', module)
```

Comment:
Review the Story name

Reply:
To be more clear, we are saying this because this component isn't a Select box.

Reply:
thanks, it should be fine now